File: springboot2.x-learning/springboot-data-jpa-multi/src/main/java/com/rtxtitanv/model/secondary/SecondaryUser.java
package com.rtxtitanv.model.secondary;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

import javax.persistence.*;

/**
 * @author rtxtitanv
 * @version 1.0.0
 * @name com.rtxtitanv.model.secondary.SecondaryUser
 * @description 从数据库用户实体类
 * @date 2020/1/7 19:32
 */
@Accessors(chain = true)
@AllArgsConstructor
@NoArgsConstructor
@Data
@Entity
@Table(name = "user")
public class SecondaryUser {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;
    @Column(name = "user_name")
    private String userName;
    @Column(name = "pass_word")
    private String passWord;
    @Column(name = "nick_name")
    private String nickName;
    @Column(name = "age")
    private Integer age;
    @Column(name = "email")
    private String email;
    @Column(name = "tel")
    private String tel;
}

File: springboot2.x-learning/springboot-data-jpa-multi-kotlin/src/main/kotlin/com/rtxtitanv/model/secondary/SecondaryUser.kt
package com.rtxtitanv.model.secondary

import javax.persistence.*

/**
 * @name com.rtxtitanv.model.secondary.SecondaryUser
 * @description 从数据库用户实体类
 * @author rtxtitanv
 * @date 2020/2/5 18:29
 * @version v1.0.0
 */
@Entity
@Table(name = "user")
data class SecondaryUser(@Id
                         @GeneratedValue(strategy = GenerationType.IDENTITY)
                         @Column(name = "id")
                         var id: Long? = null,
                         @Column(name = "user_name")
                         var userName: String? = null,
                         @Column(name = "pass_word")
                         var passWord: String? = null,
                         @Column(name = "nick_name")
                         var nickName: String? = null,
                         @Column(name = "age")
                         var age: Int? = null,
                         @Column(name = "email")
                         var email: String? = null,
                         @Column(name = "tel")
                         var tel: String? = null)